.code-samples.meilisearch.yaml

```diff
@@ -773,23 +825,25 @@ primary_field_guide_add_document_primary_key: |-
   curl \
     -X POST 'http://localhost:7700/indexes/books/documents?primaryKey=reference_number' \
     -H 'Content-Type: application/json' \
-    --data-binary '[{
+    --data-binary '[
+      {
         "reference_number": 287947,
         "title": "Diary of a Wimpy Kid",
         "author": "Jeff Kinney",
-        "genres": ["comedy","humor"],
+        "genres": [
+          "comedy",
+          "humor"
+        ],
         "price": 5.00
      }]'
```

Comment:
Shouldn't these also be separated into two different lines?

Reply:
Do you mean:

```
"price":
   5.00
```